bundles/edu.kit.kastel.sdq.eclipse.grading.api/.project

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>edu.kit.kastel.sdq.eclipse.grading.api</name>
+	<name>edu.kit.kastel.sdq.eclipse.common.api</name>
```

Comment:
It seems that the directory has not been renamed to common? Or is this a GH Bug?